content/applications/hr/recruitment.rst

```diff
@@ -1,10 +1,730 @@
+
+When the message reads :guilabel:`0 are invalid`, type in the message to be sent in the
+:guilabel:`Message` field. Beneath the message field, the number of characters, as well as the
+amount of text messages required to send the message (according to GSM7 criteria) appears. Click
+:guilabel:`Put In Queue` to have the text sent later, after any other messages are scheduled, or
+click :guilabel:`Send Now` to send the message immediately.
+
+.. image:: recruitment/send-sms.png
+   :align: center
+   :alt: Send a text message to the attendees of the meeting.
+
+.. note::
+   Sending text messages is not a default capability with Odoo. To send text messages, credits are
+   required, which need to be purchased. For more information on credits and plans, refer to
+   :doc:`this documentation <../marketing/sms_marketing/pricing/pricing_and_faq>`.
+
+Second interview
+----------------
+
+After an applicant has passed the :guilabel:`First Interview` stage, they next move to the
+:guilabel:`Second Interview` stage. To move the applicant to the next stage, drag and drop the
+applicant's card to the :guilabel:`Second Interview` stage, or click on the :guilabel:`Second
+Interview` stage button at the top of the individual applicant's card.
+
+When the applicant's card moves to the :guilabel:`Second Interview` stage, there are no automatic
+activities or emails configured for this stage, unlike for the :guilabel:`First Interview` stage.
+The recruiter should :ref:`schedule a second interview <recruitment/schedule>` with the applicant,
+following the same process as for the first interview.
+
+Contract proposal
+-----------------
+
+When the applicant has passed the interview stages and an offer is ready to be sent, they move to
+the :guilabel:`Contract Proposal` stage. Drag and drop the applicant's card to the
+:guilabel:`Contract Proposal` stage, or click on the :guilabel:`Contract Proposal` stage button at
+the top of the individual applicant's card.
+
+The next step is to send an offer to the applicant. On the applicant's card, click the
+:guilabel:`Generate Offer` smart button. A :guilabel:`Generate a Simulation Link` pop-up appears
+All fields are pre-populated with information from the job position, except for the :guilabel:`Force
+New Cars List`, :guilabel:`Default Vehicle`, and the :guilabel:`Canteen Cost` (this is set to
+`0.00`).
+
+.. note::
+   Not all fields may appear on the :guilabel:`Generate a Simulation Link` pop-up. Depending on the
+   localization setting for the company and the applications installed, some fields may not appear.
+   For example if the *Fleet* application is not installed, the fields related to vehicles will not
+   appear on the pop-up.
+
+Universal fields
+~~~~~~~~~~~~~~~~
+
+The following fields appear for all offers sent to applicants regardless of localization settings.
+
+- :guilabel:`Offer Template`: The template currently being used to populate the :guilabel:`Generate
```

Comment:
The offer template can also be accessed by going to configuration--> JOb Positions --> select the position--> recruitment tab--> Contract Template

Reply:
It's likely edits will be made manually and not on the fly with this document

Reply:
I go into depth in the other doc, so I didn't add any instructions that overlapped with making these form scratch.